frontend/src/API.js

```diff
@@ -2,6 +2,15 @@ import axios from 'axios';
 
 const API = axios.create({
 	baseURL: `${process.env.REACT_APP_SERVER_API}`,
+		timeout: 5000,
+		headers: {
+			Authorization: localStorage.getItem('access_token') 
+				? 'knox' + localStorage.getItem('access_token')
+				: null,
+				'Content-Type': 'application/json',
+				accept: 'application/json'
```

Comment:
```suggestion
			Authorization: localStorage.getItem('access_token'),
			'Content-Type': 'application/json',
			accept: 'application/json'
```
indent then you can set the access token directly, it will set to null if there is no token
You dont need condition, just append the knox when you are saving the token in your local storage